doc/tutorials/chapter_6/00_combinational_logic.md

```diff
@@ -0,0 +1,383 @@
+- The circuit contains no cyclic paths: every path through the circuit visits each circuit node at most once.
+
+## What is Conditionals?
+
+In ROHD, `Conditional` type statement must always written within a type of `_Always` block, similar to System Verilog. There are two type of `_Always` blocks: `Sequential` and `Combinational`, which map to System Verilog's `always_ff` and `always_comb`, respectively.
+
+```dart
+// Example of always_comb
+Combinational([
+  // Conditionals written inside here...
+]);
+```
+
+`Combinational` Block takes a list of `Conditional` statements. Different kinds of `Conditional` statement, such as `If`, may be composed of more `Conditional` statements. You can create `Conditional` composition chains as deep as you like.
+
+Conditional statements are executed imperatively and in order, just like the contents of `always` blocks in SystemVerilog. `_Always` blocks in ROHD map 1-to-1 with SystemVerilog `always` statements when converted.
+
+Assignments within an `_Always` should be executed conditionally, so use the `<` operator which creates a `ConditionalAssign` object instead of `<=`. The right hand side a `ConditionalAssign` can be anything that can be `put` onto a `Logic`, which includes `int`s. If you're looking to fill the width of something, use `Const` with the `fill = true`.
+
+Example below shows the example of `Combinational` with `Logic` and `ConditionalAssign`.
+
+```dart
+final and1 = carryIn & (a ^ b);
+final and2 = b & a;
+
+Combinational([
+    // notice that `<` in used instead of `<=` inside _Always
+    sum < (a ^ b) ^ carryIn,
+    carryOut < and1 | and2,
+]);
+```
+
+The most important part that you have to notice here is the assignment operator in `Combinational` Block is different from common dart operator. So, a few operators that being use commonly are:
+
+- `.eq()` function map to `==`
+- `.lt()` function map to `<`
+- `.lte()` function map to `<=`
+- `(>)` function map to `>`
+- `(>=)` function map to `>=`
+
+## If, ElseIf, Else
+
+Alright, now we know how the operator in ROHD. we can dive into the `If...Else` in ROHD. In dart, `if...else` is used as a conditional for hardware generation, we can think of it as `if` condition A filled, then generate this pieces of hardware `else` generate that pieces of hardware. While in ROHD, `If...Else` is conditionally assignment which assign signal to a port, which we can think something like `If` Logic signal `A` is present, `Then` assign output port `B` to `A`.
+
+In today tutorial, we will review how to assign value to PORT using ROHD `If...Else` conditionals. Let start by understanding ROHD `If...Else` conditionals. There are several ways of using `If...Else` in ROHD, but the most prefferable way is using `If.block` which is more readable and clean.
+
+*Note: `If...Else` and `Case` in ROHD is written in capital letter for starting of the alphabet.*
```

Comment:
"... is written with the first letter capitalized to differentiate from the built-in dart keywords and functionality"